src/basic/copy.c

```diff
@@ -1243,7 +1243,7 @@ int copy_access(int fdf, int fdt) {
         return 0;
 }
 
-int copy_rights(int fdf, int fdt) {
+int copy_rights_with_fallback(int fdf, int fdt, const char *patht) {
```

Comment:
`patht`??

Reply:
I was matching the existing convetion used in that function: `fdt` → `patht`. If the second path was needed, it'd be `fdf` → `pathf`. I would like less cryptic names, maybe `fd_from`, `fd_to`, `path_from`, `path_to`, but then I guess the whole file should be changed.

Reply:
Ah, I see. Thanks.